frog/imports/client/Wiki/index.js

```diff
@@ -617,13 +768,36 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
       });
     }
 
-    const secondaryNavItems = [
-      {
-        title: 'Restore deleted page',
-        icon: RestorePage,
-        callback: () => this.openRestorePageModal(invalidPages)
-      }
-    ];
+    const secondaryNavItems = this.state.isOwner
+      ? [
+          {
+            title: 'Restore deleted page',
+            icon: RestorePage,
+            callback: () => this.openRestorePageModal(invalidPages)
+          },
```

Comment:
This is not DRY. Make a push to `secondaryNavItems` instead of repeating the definition `'Restore deleted page'`

Reply:
Ok, fair point